test/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -306,15 +252,15 @@ public void testParseTime_with_DST() throws Exception {
 		TimeZone.setDefault(dstTimezone);
 
 		String time = "12:03:05.000Z";
-		testTime(time, 43385000L);
+		testTime(time, 32585000);
```

Comment:
Does the changed value here mean that we have changed the behavior in unwanted ways?